### PR TITLE
Update running Chihaya documentation for v2

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

File: quay-enterprise/running-chihaya.md [hunk 1]
```diff
@@ -54,8 +43,8 @@ the reachable endpoint for the Quay Enterprise instance and the tracker itself,
 Run the following commands to start Chihaya under a Docker container with the specified configuration mounted, making sure to point the `config.json` to the file created above.
 
 ```sh
-$ docker pull quay.io/jzelinskie/chihaya:v1.0.1
-$ docker run -p 6880-6882:6880-6882 -v $PWD/config.json:/config.json:ro quay.io/jzelinskie/chihaya:v1.0.1 -v=5
+$ docker pull quay.io/jzelinskie/chihaya-git:2.0.0-rc.1
```
Comment (jzelinskie) [inline quay-enterprise/running-chihaya.md]: quay.io/jzelinskie/chihaya:v2.0.0-rc.1

File: quay-enterprise/running-chihaya.md [hunk 2]
```diff
@@ -54,8 +43,8 @@ the reachable endpoint for the Quay Enterprise instance and the tracker itself,
...
+$ docker run -p 6880-6882:6880-6882 -v $PWD/config.yaml:/config.yaml:ro quay.io/jzelinskie/chihaya-git:2.0.0-rc.1 -v=5
```
Comment (jzelinskie) [inline quay-enterprise/running-chihaya.md]: remove the `-v=5` flag

and mount to `/etc/chihaya.yaml:ro`

Comment (josephschorr) [inline quay-enterprise/running-chihaya.md]: Updated

File: quay-enterprise/running-chihaya.md (hunk 1 above)
Comment (josephschorr) [inline quay-enterprise/running-chihaya.md]: Updated

File: quay-enterprise/running-chihaya.md [hunk 3]
```diff
@@ -54,8 +43,8 @@ the reachable endpoint for the Quay Enterprise instance and the tracker itself,
 Run the following commands to start Chihaya under a Docker container with the specified configuration mounted, making sure to point the `config.json` to the file created above.
 
 ```sh
-$ docker pull quay.io/jzelinskie/chihaya:v1.0.1
-$ docker run -p 6880-6882:6880-6882 -v $PWD/config.json:/config.json:ro quay.io/jzelinskie/chihaya:v1.0.1 -v=5
+$ docker pull quay.io/jzelinskie/chihaya:v2.0.0-rc.1
+$ docker run -p 6880-6882:6880-6882 -v $PWD/config.yaml:/etc/config.yaml:ro quay.io/jzelinskie/chihaya:v2.0.0-rc.1
```
Comment (jzelinskie) [inline quay-enterprise/running-chihaya.md]: `chihaya.yaml` not `config.yaml`

**Comment by josephschorr** [comment]
@jzelinskie: Updated

File: quay-enterprise/running-chihaya.md [hunk 4]
```diff
@@ -6,56 +6,45 @@ The <a href="https://github.com/chihaya/chihaya">Chihaya</a> project is an open
...
+      audience: '{TRACKER LOCATION}/announce'
+      jwk_set_url: '{QE LOCATION}/keys/services/quay/keys'
+      jwk_set_update_interval: 5m
 ```
 
 
 ## Running
 
-Run the following commands to start Chihaya under a Docker container with the specified configuration mounted, making sure to point the `config.json` to the file created above.
+Run the following commands to start Chihaya under a Docker container with the specified configuration mounted, making sure to point the `chihaya.json` to the file created above.
```
Comment (jzelinskie) [inline quay-enterprise/running-chihaya.md]: s/json/yaml

**Pull request merged**